server/game/cards/14-HCWM/PeteSpence.js

```diff
@@ -0,0 +1,55 @@
+const DudeCard = require('../../dudecard.js');
+const GameActions = require('../../GameActions/index.js');
+const StandardActions = require('../../PlayActions/StandardActions.js');
+
+class PeteSpence extends DudeCard {
+    setupCardAbilities(ability) {
+        this.job({
+            title: 'Pete Spence',
+            playType: 'noon',
+            cost: ability.costs.bootSelf(),
+            target: {
+                activePromptTitle: 'Mark an in-town location other than Pete\'s current location',
+                cardCondition: {
+                    location: 'play area',
+                    controller: 'any',
+                    condition: card => !card.isOutOfTown() && card.gamelocation !== this.gamelocation
+                },
+                cardType: ['location', 'townsquare', 'outfit']
+            },
+            message: context =>
+                this.game.addMessage('{0} plays {1} on {2}.', context.player, this, context.target),
+            onSuccess: (job, context) => {
+                this.game.resolveGameAction(
+                    GameActions.search({
+                        title: 'Select an Attire or Non-Unique Weapon for Pete to fetch',
```

Comment:
Again, this can be shorter. To be clear what is the prompt used for, we can add `source: this` to the properties.
```suggestion
                        title: 'Select an Attire or Non-Unique Weapon',
```

Reply:
:heavy_check_mark: